src/ui/api/repository.go

```diff
@@ -605,6 +605,43 @@ func (ra *RepositoryAPI) GetTopRepos() {
 	ra.ServeJSON()
 }
 
+// Put updates description info for the repository
+func (ra *RepositoryAPI) Put() {
+	name := ra.GetString(":splat")
+	repository, err := dao.GetRepositoryByName(name)
```

Comment:
Is RepoName enough? I think also need ProjectID to locate the repository.

Reply:
The first part of the repository name is the project name, so the repository name is unique.